fcrepo-connector-file/src/main/java/org/fcrepo/connector/file/FedoraFileSystemConnector.java

```diff
@@ -324,12 +322,10 @@ protected void touchParent( final String id ) {
 
     /* Overriding so unit test can mock. */
     @Override
-    @VisibleForTesting
```

Comment:
Yes, I find them helpful from an informational perspective. `public` and `protected` methods mean increased responsibility. It is useful to know that some methods have been relaxed from `private` for the sake of unit testing. 
Instead of simply removing these annotations, I would suggest replacing them with a prominent comment/javadoc that communicates the same idea.